pkg/repository/udmrepo/kopialib/backend/azure/azure_storage.go

```diff
@@ -0,0 +1,308 @@
+// Package azure implements Azure Blob Storage.
```

Comment:
I didn't get how `Velero needs the capability to interact with the repository concurrently while Kopia doesn't` is solved, could you share some more insights?